File: pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;
import entity.Result;

import java.util.List;

public interface BrandService {
    public List<TbBrand>findAll();

    //数据分页显示
    public PageResult findPage(Integer pageNum,Integer pageSize);
    //商品增加
    public void add(TbBrand tbBrand);
    //商品修改
    public void update(TbBrand tbBrand);
    public TbBrand findOne(Long id);//根据ID查询需要修改商品的数据
    //商品删除
    public void delete(Long ids[]);
    //查询后分页显示
    //brand 传入的查询条件
    public PageResult findPage(TbBrand brand,Integer pageNum,Integer pageSize);

}

File: pinyougou_manager_web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {
    this.findAll=function () {
        return   $http.get('../brand/findAll.do');
    }
    this.findPage=function (pageNum,pageSize) {
        return $http.get('../brand/findPage.do?page='+pageNum+"&rows="+pageSize);
    }
    this.add=function (entity) {
        return $http.post('../brand/add.do',entity)
    }
    this.update=function (entity) {
        return $http.post('../brand/update.do',entity)
    }
    this.findOne=function (id) {
        return $http.get('../brand/findOne.do?id='+id)
    }
    this.dele=function (ids) {
        return $http.get('../brand/delete.do?ids='+ids)
    }
    this.search=function (pageNum,pageSize,searchEntity) {
        return $http.post('../brand/search.do?page='+pageNum+"&rows="+pageSize,searchEntity)
    }
})
